README.md

```diff
@@ -1,3 +1,4 @@
+[![Build Status][travis-image]][travis-url] [![Coverage Status][coveralls-image]][coveralls-url]
```

Comment:
We still don't have a coveralls account/set up. I don't think this would be harmful, but not needed at this point